libs/perun/services/src/lib/store.service.ts

```diff
@@ -83,7 +102,6 @@ export class StoreService {
           currentValue = this.defaultConfig[keys[i]];
         } else {
           if (currentValue === undefined) {
-            console.error('Missing value in default config: ' + keys);
```

Comment:
Well, this log was there for a reason. I think it should print some error if some default value is not specified, since it almost certainly shows that we have some bug in the code. I just didn't want it to print error, when the branding is not found, since that is OK and we expect it to happen.